src/main/java/com/example/bookstore/controller/login/LoginsController.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstore.controller.login;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/login")
+public class LoginsController {
+    @GetMapping("/showLoginPage")
+    public String showLoginPage() {
+        return "login/form";
```

Comment:
```suggestion
        return "sessions/loginForm";
```
loginForm hay login đều dc